integration/client_sdk/go/utils.go

```diff
@@ -97,3 +115,158 @@ func setupNetwork(channel string) (*gateway.Network, error) {
 
 	return network, nil
 }
+
+func setup(ccID, ccPath string) error {
+	nwDetails := &testutils.NetworkDetails{
+		ChannelID: "mychannel",
+		Peers:     []string{"jdoe_test.sampleorg.example.com"},
+		Orderers:  []string{"orderer.example.com"},
+	}
+
+	mrenclave, err := fpc.ReadMrenclave(ccPath)
+	if err != nil {
+		return err
+	}
+
+	ccDetails := &testutils.ChaincodeDetails{
+		Id:           ccID,
+		Path:         ccPath,
+		Version:      mrenclave,
+		Lang:         fpc.ChaincodeLang,
+		Policy:       policydsl.SignedByAnyMember([]string{"SampleOrg"}),
```

Comment:
Oh you made a good point here. Since this code is integration test only and the current setting the actual policy picked here is not so important (functional), however, as this integration test may also function as documentation and as an example how to use FPC this *anyMember* policy is certainly not good. ACK.

Here in this test we can just unset the policy and thereby use the default implicit policy. But I see a problem here that we do not document this in code properly.  We could provide a `fpcpolicy.MajorityOfMembers(members []string),` function that we use throughout our tests. Thereby by reading our code it is clear that we always use a "strong" policy.

Reply:
Hmm, i don't think you can implement in a robust way with current fabric a MajorityOfMembers. Per-chaincode fabric allows only signature policies; majority can only be defined in configtx.yaml and alike.   Note as (number of) orgs might change over time, you won't be able to define a robust majority based on signature policies!
In any case, this is the same instance of the issue we discussed last week for the integration-test cli and i thought the conclusion was that we just add a corresponding comment?

Reply:
yes, you are right. In our discussion yesterday with Bruno we also agreed that my proposed `MajorityOfMembers` function is not strong enough for dynamic membership on a channel. I added some more comments on this function and devoted it for testing only.